titanic.py

```diff
@@ -134,23 +127,15 @@ def run_demo():
 
     sorted_scores = sorted(scores.items(), key=lambda pair: pair[1], reverse=True)
     dataset = dataset[[column[0] for column in sorted_scores[0:keep]] + ["survived"]]
-    features = list(set(dataset.columns).difference(('survived',)))
-    features = sorted(features)
+    features = sorted(list(set(dataset.columns) - {'survived'}))
 
-    # Build a QUBO that maximizes MI between survival and a subset of features
```

Comment:
Might be good to leave this comment in for clarity.

Reply:
Done